apps/judicial-system/backend/src/app/modules/case-table/whereOptions/publicProsecutor.ts

```diff
@@ -0,0 +1,43 @@
+import { Op } from 'sequelize'
+
+import {
+  CaseIndictmentRulingDecision,
+  EventType,
+  indictmentCases,
+  type User as TUser,
+} from '@island.is/judicial-system/types'
+
+import { buildEventLogExistsCondition } from './conditions'
+
+// public prosecutor indictments
+// specific for prosecutors at the public prosecutor office
+
+const publicProsecutorIndictmentSharedWhereOptions = (user: TUser) => ({
+  is_archived: false,
+  type: indictmentCases,
+  indictment_ruling_decision: [
+    CaseIndictmentRulingDecision.FINE,
+    CaseIndictmentRulingDecision.RULING,
+  ],
+  [Op.and]: [
```

Comment:
For added integrity please consider adding a check to make sure the case is in a completed state.